handsontable/src/plugins/manualRowResize/manualRowResize.js

```diff
@@ -351,9 +351,9 @@ export class ManualRowResize extends BasePlugin {
    * @returns {boolean}
    */
   checkIfRowHeader(element) {
-    const thElement = closest(element, ['TH'], this.hot.rootElement);
+    const tbody = closest(element, ['TBODY'], this.hot.rootElement);
 
-    return thElement && element.parentNode?.parentNode?.tagName === 'TBODY';
+    return tbody === this.hot.view._wt.wtOverlays.inlineStartOverlay.clone.wtTable.TBODY;
```

Comment:
This condition makes the resize handle not appear when the row headers are in the corner overlay (overlapping fixed rows and columns)

Reply:
Thanks for spotting that. I fixed and covered the change within https://github.com/handsontable/handsontable/pull/10650/commits/e8be061f7b55a3c90e058a284dadd9e3a3c9fa59.